content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,115 @@
+
+The `builder` is meant to help developers to quickly assemble and build their own Collector's distributions, based on a given configuration file.
+
+As part of the process the `builder` will generate the Collector's source code, which you can borrow and steal as your own to help build and debug your own components, so let's get started.
+
+## Step 1 - Install the builder
+
+You can install the builder through the `go install` command. 
+
+Open your terminal and type:
+
+```
+go install go.opentelemetry.io/collector/cmd/builder@latest
+```
+
+The `builder` command will be installed on your `$GOPATH/bin`, which most likely is already added to your path.
+
+To make sure it's there, go to your terminal and type `builder help`, and once you hit enter you should have the following output:
+
+```
+
+OpenTelemetry Collector distribution builder (dev)
+
+Usage:
+  builder [flags]
+  builder [command]
+
+Available Commands:
+  completion  Generate the autocompletion script for the specified shell
+  help        Help about any command
+  version     Version of opentelemetry-collector-builder
+
+Flags:
+      --config string            config file (default is $HOME/.otelcol-builder.yaml)
+      --description string       A descriptive name for the OpenTelemetry Collector distribution (default "Custom OpenTelemetry Collector distribution")
+      --go string                The Go binary to use during the compilation phase. Default: go from the PATH
+  -h, --help                     help for builder
+      --module string            The Go module for the new distribution (default "go.opentelemetry.io/collector/cmd/builder")
+      --name string              The executable name for the OpenTelemetry Collector distribution (default "otelcol-custom")
+      --otelcol-version string   Which version of OpenTelemetry Collector to use as base (default "0.42.0")
+      --output-path string       Where to write the resulting files (default "/var/folders/86/s7l1czb16g124tng0d7wyrtw0000gn/T/otelcol-distribution3618633831")
+      --skip-compilation         Whether builder should only generate go code with no compile of the collector (default false)
+      --version string           The version for the OpenTelemetry Collector distribution (default "1.0.0")
+
+Use "builder [command] --help" for more information about a command.
+```
+
+## Step 2 - Create a builder config file
+
+The builder config file is a `yaml` where you basically pass information about the code generation and compile process combined with the components that you would like to add to your Collector's distribution.
+
+Go ahead and create a `builder-config.yaml` with the following content:
+
+> builder-config.yaml
+```yaml
+dist:
+    module: github.com/open-telemetry/opentelemetry-collector # the module name for the new distribution, following Go mod conventions. Optional, but recommended.
```

Comment:
This is the distribution's module name. It should not be using the same name as our official distributions.

Reply:
I missed that one, I will fix it in the next commit